demos/http/common/include/http_demo_utils.h

```diff
@@ -26,6 +26,9 @@
 /* Transport interface include. */
 #include "transport_interface.h"
 
+/* HTTP API header. */
```

Comment:
This file is copied from https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1318 and not the responsibility of that PR.